sdk/communication/azure-communication-administration/src/samples/java/com/azure/communication/administration/ReadmeSamples.java

```diff
@@ -108,4 +117,218 @@ public void deleteUser() {
             e.printStackTrace();
         }
     }
+
+    /**
+     * Sample code for creating a sync Phone Number Client.
+     *
+     * @return the Phone Number Client.
+     * @throws NoSuchAlgorithmException if Communication Client Credential HMAC not available
+     * @throws InvalidKeyException if Communication Client Credential access key is not valid
+     */
+    public PhoneNumberClient createPhoneNumberClient()
+        throws NoSuchAlgorithmException, InvalidKeyException {
+        // Your can find your endpoint and access token from your resource in the Azure Portal
```

Comment:
Nit: s/Your/You

Reply:
Got copied over incorrectly, will fix it.